.......FF....
======================================================================
FAIL: test_jam_e2e_scan_twas (__main__.LearningCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/jam/wca4java/tests/jamJarTestsE2E_O4.py", line 19, in test_jam_e2e_scan_twas
    self.assertTrue(filecmp.cmp('/root/jam/wca4java/tests/actual/e2e/scan_twas_ruleid.txt', '/root/jam/wca4java/tests/expected/e2e/scan_twas_ruleid.txt'))
AssertionError: False is not true

======================================================================
FAIL: test_jam_e2e_scan_twas_2 (__main__.LearningCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/jam/wca4java/tests/jamJarTestsE2E_O4.py", line 37, in test_jam_e2e_scan_twas_2
    self.assertTrue(filecmp.cmp('/root/jam/wca4java/tests/actual/e2e/scan_twas_ruleid_2.txt', '/root/jam/wca4java/tests/expected/e2e/scan_twas_ruleid_2.txt'))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 13 tests in 0.003s

FAILED (failures=2)
